rcl_logging_spdlog/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rcl_logging_spdlog</name>
+  <version>0.1.0</version>
+  <description>C API providing common interface to a shared library wrapping 3rd party loggers.</description>
+  <maintainer email="clalancette@openrobotics.org">Chris Lalancette</maintainer>
+  <author email="clalancette@openrobotics.org">Chris Lalancette</author>
+  <license>MIT</license>
```

Comment:
The files added have an Apache 2.0 license.
```suggestion
  <license>Apache License 2.0</license>
```

Reply:
Good call, will fix.